### ISSUE TITLE
Duplicate notifications on slack

### ISSUE THREAD

**Issue by brad-decker** [wavebox/waveboxapp#271]
* Which version of Wavebox are you using? 3.1.10

* Which Operating System are you using? Linux, elementary os loki

I get multiple sets of notifications constantly with slack. I haven't noticed the same issue with inbox. Its almost always 2 but occasionally its 4.

**Comment by benwgarton** [comment]
Same Slack windows 10 ver. 3.1.10 double notification banners.

**Comment by Thomas101** [comment]
Do the notifications have the exact same text or are they different in any way? Does the behaviour change depending on what you're being notified about? (e.g. a DM vs a mention?)

**Comment by brad-decker** [comment]
The only time behavior is changed is if i'm in the active Dm window when a message comes in. in that instance I receive only 1 notification (would expect 0).

**Comment by Thomas101** [comment]
How many slack accounts do you have added in total? Could you give the following a try?

1.) Make sure that your web-browser is fully quit (chrome, firefox, edge etc...) and try re-sending a notification.
2.) Right click on the account in the left and go to account settings. Then toggle the Show Notifications button and retry. How many do you receive?
3.) Right click on the account in the left and go to Inspect. Then click on the console tab and paste the following... `new Notification('test')`. Does that bring up a notification?

Thanks

**Comment by brad-decker** [comment]
1) With chrome exited i had a team mate message me and received 2 notifications of his message
2) Toggled off and on and clicked test notification -- only received one -- asked a teammate to message me and received 2. 
3) Pasting that into the console did not trigger a notification.

**Comment by brad-decker** [comment]
also only have one slack account

**Comment by Thomas101** [comment]
I wonder if there's some erroneous processes running here. Could you try running `ps auwx` from the terminal to get the process list? Feel free to send this to support [at] wavebox.io rather uploading it here.

**Comment by mikeupton** [comment]
I am using Wavebox on Windows 10 with a single slack account and having the same issue of two notifications for each slack message.

**Comment by Thomas101** [comment]
@mikeupton do both come up at the same time or is there a delay between them? If at the same time would you be able to send a screenshot across?

**Comment by mikeupton** [comment]
They come up about 2 seconds apart.

**Comment by Thomas101** [comment]
Thanks, we've managed to reproduce the issue our end...

1.) Add slack account
2.) Disconnect internet for extended period -- this will leave two websockets open
3.) Reconnect

**Comment by Thomas101** [comment]
Hi, we've put a fix out for this in the beta channel. There's some instructions [here](https://wavebox.io/kb/beta-channel) on how to switch across so you can give it a try

**Comment by mikeupton** [comment]
I downloaded your fix last week and I think it worked.  However I just took
a new update - and I think it regressed to previous behavior..

**Comment by Thomas101** [comment]
@mikeupton we've just pushed 3.1.16. Would you be able to try with that? Thanks

**Comment by mikeupton** [comment]
This did not fix the issue.

**Comment by Thomas101** [comment]
@mikeupton can you go the View Menu and then Developer Tools. Then click on the (1) Network tab and the (2) WS filter (see attached screenshot).

![screen shot 2017-07-25 at 11 09 57](https://user-images.githubusercontent.com/103586/28567495-55c90b38-712a-11e7-9c16-6a2cf1c89fe9.png)

in the section labelled (3) how many requests do you see? There should be one which is the Wavebox Socket and another one for slack

**Comment by Thomas101** [comment]
Hi, this has just been released into the stable channel of Wavebox. You can get the update by using the Application/File menu and clicking `Check for Update` or by downloading the latest version from our [download page](https://wavebox.io/download). Thanks!

**Comment by jubram** [comment]
Hey, @Thomas101 ! I know this issue has been closed for a while now.

I've been using Wavebox for about 2 years now. I've just updated for the newest Wavebox 10 and this issue reapered for me...

I have only 1 slack account connected to my Wavebox.

I'm using Ubuntu Budgie:
![image](https://user-images.githubusercontent.com/8689118/73951012-d28e9080-48f4-11ea-8d0c-fb833940163d.png)


The network tab shows me this:
![image](https://user-images.githubusercontent.com/8689118/73950877-9ce9a780-48f4-11ea-8a1d-2c69aae23638.png)

The two notifications come at the same time:
![image](https://user-images.githubusercontent.com/8689118/73950902-a8d56980-48f4-11ea-9a9b-f35b62010901.png)

Thanks!
